README.md

```diff
@@ -181,7 +111,7 @@ above and use the following option for building with nGraph:
 
         cd tensorflow
         git checkout v1.12.0
```

Comment:
Do we need to talk about the ./configure options
Say N to  Cuda etc.
And -march=broadwell ?

Reply:
May be move option 2 -step 3 to option 3

Reply:
We might also need to add `https://github.com/tensorflow/tensorflow.git` before the `cd tensor flow`

Reply:
oh, its not `v` any more, its `r`:
`v1.12.0`->`r1.12`

Reply:
It's embedded in the Python build step. Don't want to talk about it here.